src/Features/Core/Portable/CodeRefactorings/CodeRefactoringService.cs

```diff
@@ -132,16 +134,21 @@ static ProjectCodeRefactoringProvider.ExtensionInfo GetExtensionInfo(ExportCodeR
                     if (priority != CodeActionRequestPriority.None && priority != provider.RequestPriority)
                         continue;
 
-                    tasks.Add(Task.Run(() =>
+                    tasks.Add(Task.Run(async () =>
                         {
+                            const int CodeRefactoringTelemetryDelay = 500;
```

Comment:
Can you add a comment about this delay? What purpose does it server and how did you come to this specific constant `500`?

Reply:
wfm.

Reply:
note: we still may want to both put this constant in a more uniform plce.